### ISSUE TITLE
Change app colour

### ISSUE THREAD

**Issue by ErmanSayin** [ErmanSayin/Themely#113]

**Comment by papermario69** [comment]
If you're looking for suggestions, blue might be good considering most people go through Homebrew to get to it.?

**Comment by ErmanSayin** [comment]
Title was too vague, sorry

**Comment by papermario69** [comment]
Yeah an option to change color would be good.

**Comment by Juanito9001** [comment]
I agree, an option to change the colour would be good, and maybe change the font of the words, it can be more attractive for the users who wants personalyze the style of the app
